docs/source/api_docs/internals.rst

```diff
@@ -38,11 +38,11 @@ can be used.
 
 A `Multipoint` is a group of points, and is the second simplest GeoArrow 
 geometry type. It is identical to points, with the addition of a 
-`multipoints_offsets` buffer. The offsets buffer stores N+1 indices. The
+``multipoints_offsets`` buffer. The offsets buffer stores N+1 indices. The
```

Comment:
Why the double back quotes? Markdown only requires single. Is this a Sphinx thing?

Reply:
This isn't markdown but reStructuredText (we don't like that and cuDF is converting all documentation to markdown). Per [reST doc](https://docutils.sourceforge.io/docs/user/rst/quickref.html#inline-markup), only double backticks is recognized as monospaced fonts and single backtick is application specific. In Sphinx, double backticks are rendered as inline highlighted code:
![image](https://user-images.githubusercontent.com/13521008/168697837-d455f550-8640-44aa-87b1-b15cf98cb7a5.png)
And single backtick are rendered as italics:
![image](https://user-images.githubusercontent.com/13521008/168697928-e795c086-b871-4364-962b-313fd7b453ff.png)

Reply:
> (we don't like that and cuDF is converting all documentation to markdown)

So shouldn't we do the same in cuSpatial?

Reply:
I haven't investigated how many hurdles it takes to do all the conversion but would love to tackle that as a follow up to this. Meanwhile, perhaps the author of https://github.com/rapidsai/cudf/pull/10691 can explain a bit on this :) @shwina